src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -49,8 +52,8 @@
       />
       <BalBtn
         v-if="trading.isLoading.value"
-        loading
-        disabled
+        :loading="true"
+        :disabled="true"
```

Comment:
You don't need to pass `true` here, simply setting the attribute is the same thing, `loading` is equal to `:loading="true"`

Reply:
oh oops, you are right.